src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -685,44 +687,72 @@ function saturation_adjustment_NewtonsMethod(e_int::FT, ρ::FT, q_tot::FT) where
       T -> internal_energy_sat(T, ρ, q_tot) - e_int,
       T_ -> ∂e_int_∂T(T_, e_int, ρ, q_tot),
       T_1,
-      NewtonsMethod(), CompactSolution(), FT(1e-3), 10)
-      if !sol.converged
-        error("saturation_adjustment_NewtonsMethod did not converge")
-      end
+      NewtonsMethod(), CompactSolution(), tol, maxiter)
+    if !sol.converged
+      error("saturation_adjustment did not converge")
+    end
     return sol.root
   end
 end
 
 """
-    saturation_adjustment(e_int, ρ, q_tot)
+    ΔT_min(::Type{FT})
+
+Minimum interval for saturation adjustment using Secant method
+"""
+@inline ΔT_min(::Type{FT}) where FT = FT(3)
+
+"""
+    ΔT_max(::Type{FT})
+
+Maximum interval for saturation adjustment using Secant method
+"""
+@inline ΔT_max(::Type{FT}) where FT = FT(10)
+
+"""
+    bound_upper_temperature(T_1::FT, T_2::FT) where {FT<:Real}
```

Comment:
Remove unless we still need it with the new tests.

Reply:
This is still used in `saturation_adjustment_SecantMethod` and `saturation_adjustment_q_tot_θ_liq_ice`